packages/react-magma-dom/src/components/Select/shared.ts

```diff
@@ -91,19 +107,18 @@ export const StyledList = styled('ul')<{ isOpen?: boolean; maxHeight: string }>`
 export const StyledItem = styled('li')<{
   isInverse?: boolean;
   isFocused?: boolean;
+  isDisabled?: boolean;
 }>`
   align-self: center;
   background: ${props => buildListHoverColor(props)};
   border: 2px solid;
   border-color: ${props => buildListFocusColor(props)};
   cursor: default;
-  color: ${props =>
-    props.isInverse
-      ? props.theme.colors.neutral100
-      : props.theme.colors.neutral700};
+  color: ${props => buildListItemColor(props)};
   line-height: 24px;
   margin: 0;
   padding: 8px 16px;
+  cursor: ${props => (props.isDisabled ? 'not-allowed' : 'pointer')};
```

Comment:
@orion-cengage do you want to use the `pointer` hand or `default` which is the existing behavior?

Reply:
I would like us to use whatever we're using on disabled buttons and other disabled items. Might be `cursor: not-allowed;`

Reply:
@orion-cengage I meant for enabled items (I agree with not allowed for disabled items). [Existing behavior](https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/select--default) is `cursor` but this PR changes it to `pointer`

Reply:
This has not been addressed yet ^

Reply:
Let's stick with the default. There doesn't seem to be an agreed-upon standard in other design systems. Some use the hand, some use the pointer. The native select uses the default, so let's just stick with that.